doc/iterations/2024.5/shapes/create_representation_using_content_of_another_representation.adoc

```diff
@@ -0,0 +1,93 @@
+= Create a representation using the content of another representation as input
+
+== Context
+
+Representation event processors are isolated on purpose, but now we want to synchronize the state of a representation event processor with another one.
+
+== Key Result
+
+We want:
+
+- To compute a representation in a representation event processor with the sate of another representation as input (e.g. a form using a diagram as input)
+- To have both states synchronized, we don't want to have only the initial state, we want both representations to be refreshed in case something occurs on one of them. 
+For example, interacting with the form should refresh the diagram and the other way around.
+- To be able to execute every existing event handler on both representations.
+- To be able to reuse frontend support for both representations easily
+- To be able to reuse the GraphQL API to interact with both representations easily
+
+== Solution
+
+Given the fact that we want to reuse the frontend, the GraphQL API and the event handlers of both representations, we want to reuse their representation event processors.
+
+The question now becomes, how to retrieve the state of a representation event processor in another one and how to synchronize this state.
+
+=== Synchronize the state
+
+Synchronizing the state is easier than retrieving it because we already have the `ChangeDescription` support which is made for that. 
+
+In our example, both `FormEventProcessor` and `DiagramEventProcessor` could be refreshed simultaneously by having some specific parameter in `ChangeDescription`.
+
+The refresh policy can also be configured by an `IRepresentationRefreshPolicyProvider`, which would allow a specific representation to be refreshed on events that don't trigger a refresh on it usually (e.g. a new form representation could be refreshed by diagram events in addition to the regular semantic changes).
+
+=== Retrieve the current state of a representation
+
+The `FormDescription#getVariableManagerInitializer` API allows to initialize variables for a given form representation.
+
+We can leverage this form-only (for now) mechanism to retrieve the `IRepresentationEventProcessor` of the representation we want the state of:
```

Comment:
This sentence is difficult to read. May be a `,` a comma is missing somewhere, or may be a word is missing but I don't understand the `... the state of:` part at the end.

Reply:
Indeed the sentence wasn't clear, I tried to rephrase it, let me know if it needs some more work